doc/amazon_sagemaker_debugger.rst

```diff
@@ -66,7 +66,7 @@ Specifying configurations for collections
 Collection Name
 ~~~~~~~~~~~~~~~
 
-``name`` in ``CollectionConfig`` is used to specify the name of the tensor collection you wish to emit and store. This name is used by SageMaker Debugger to refer to all the tensors in this collection. You can supply any valid string for the collection name. In addition to this, there are certain "built-in" collections your can emit by specifying particular collection names recognized by the hook. Examples of these collections are "gradients", "weights", "biases" etc. A full list is available at `SageMaker Debugger Built-in Collections <https://github.com/awslabs/sagemaker-debugger/blob/master/docs/api.md#built-in-collections>`__.
+``name`` in ``CollectionConfig`` is used to specify the name of the tensor collection you wish to emit and store. This name is used by SageMaker Debugger to refer to all the tensors in this collection. You can supply any valid string for the collection name. In addition, there are "built-in" collections, whose names are recognized by the hook, that you can emit simply by specifying their name. Examples of these collections are "gradients", "weights", "biases" etc. A full list is available at `SageMaker Debugger Built-in Collections <https://github.com/awslabs/sagemaker-debugger/blob/master/docs/api.md#built-in-collections>`__.
```

Comment:
* "name" should be "names" in "specifying their name" (3rd sentence)
* comma before "etc."

Reply:
Added.